openid-sharedsignals-eventdef-specification-1_0.md

```diff
@@ -0,0 +1,282 @@
+   "title":"Session Revoked",
+   "description":"Session Revoked signals that the session identified by the subject has been revoked. The explicit session identifier may be directly referenced in the subject or other properties of the session may be included to allow the receiver to identify applicable sessions.",
+   "type":"object",
+   "properties":{
+      "initiating_entity":{
+         "description":"Describes the entity that invoked the event.",
+         "type":"string",
+         "oneOf":[
+            {
+               "const":"admin",
+               "description":"an administrative action triggered the event"
+            },
+            {
+               "const":"user",
+               "description":"an end-user action triggered the event"
+            },
+            {
+               "const":"policy",
+               "description":"a policy evaluation triggered the event"
+            },
+            {
+               "const":"system",
+               "description":"a system or platform assertion triggered the event"
+            }
+         ]
+      },
+      "reason_admin":{
+         "type":"object",
+         "properties":{
+            "en":{
+               "type":"string"
+            }
+         }
+      },
+      "reason_user":{
+         "type":"object",
+         "properties":{
+            "en":{
+               "type":"string"
+            },
+            "es-410":{
+               "type":"string"
+            }
+         }
+      },
+      "event_timestamp":{
+         "type":"number"
+      }
+   }
+}
+~~~
+
+## Properties 
+
+The top level "properties" key contains an object where each property represents a key in the JSON data that’s being validated. Within each property, all normative requirements and non-normative details are specified by using the vocabulary in {{JSONSchemaValidation}}.
+
+The following is a non-normative example of a "properties" object for a CAEP event schema.
+~~~json
+{
+   "properties": {
+    "initiating_entity": {
+      "description": "Describes the entity that invoked the event.",
+      "type": "string",
+      "oneOf": [
+        {
+          "const": "admin",
+          "description": "an administrative action triggered the event"
+        },
+        {
+          "const": "user",
+          "description": "an end-user action triggered the event"
+        },
+        {
+          "const": "policy",
+          "description": "a policy evaluation triggered the event"
+        },
+        {
+          "const": "system",
+          "description": "a system or platform assertion triggered the event"
+        }
+      ]
+    },
+    "required": [ "initiating_entity" ]
+   }
+}
+~~~
+# Registry
+
+This section serves as a registry for the schemas of all registered SSF Event Types.
+
+| Event Type | Schema URI | Description
+|------|-------------|
+| CAEP  | <schema uri here> | brief description |
+| RISC  |  <schema uri here>  | brief description |
```

Comment:
hoping we can eliminate the need to classify things as "CAEP" or "RISC". If it needs to be done, it can be done via the event URI.

Reply:
Updated to Event Name